paddle/fluid/operators/optimizers/lamb_op.h

```diff
@@ -125,13 +146,24 @@ struct SparseLambMomentUpdateFunctor {
     T mom1 = moment1_[i];
     T mom2 = moment2_[i];
     T p = param_[i];
+    T beta1_pow = *beta1_pow_;
+    T beta2_pow = *beta2_pow_;
+    T mom1_unbiased, mom2_unbiased;
 
     mom1 = beta1_ * mom1 + (1 - beta1_) * g;
     mom2 = beta2_ * mom2 + (1 - beta2_) * g * g;
 
     moment1_out_[i] = mom1;
     moment2_out_[i] = mom2;
-    trust_ratio_div_[i] = mom1 / (sqrt(mom2) + epsilon_) + weight_decay_ * p;
+
+    mom1_unbiased = mom1 / (1 - beta1_pow);
```

Comment:
Write `T mom1_unbiased = mom1 / (1 - beta1_pow)' then you can do less constructing for `T` , same for `mom2_unbiased`

Reply:
Thanks, done.